### ISSUE TITLE
Bottom bar during develop build

### ISSUE THREAD

**Issue by MattixNow** [gatsbyjs/gatsby#24630]
## Description

A bottom bar appear during development build
![image](https://user-images.githubusercontent.com/32040951/83310314-e5aece80-a20b-11ea-8f3e-560dceaf8a5e.png)

### Steps to reproduce

`yarn develop`

### Expected result

No bottom bar should be better for user, we cant see the output

### Actual result

Result hidden

### Environment

```bash

  System:
    OS: Windows 10 10.0.18363
    CPU: (12) x64 Intel(R) Core(TM) i7-8700 CPU @ 3.20GHz
  Binaries:
    Node: 12.16.2 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.21.1 - ~\.yarn\bin\yarn.CMD
    npm: 6.14.4 - C:\Program Files\nodejs\npm.CMD
  Languages:
    Python: 2.7.18 - /c/Python27/python
  Browsers:
    Edge: 44.18362.449.0
  npmPackages:
    gatsby: ^2.21.37 => 2.21.37 
    gatsby-image: ^2.4.4 => 2.4.4 
    gatsby-plugin-manifest: ^2.4.5 => 2.4.5 
    gatsby-plugin-offline: ^3.2.3 => 3.2.3 
    gatsby-plugin-react-helmet: ^3.3.1 => 3.3.1 
    gatsby-plugin-sharp: ^2.6.4 => 2.6.4 
    gatsby-source-filesystem: ^2.3.4 => 2.3.4 
    gatsby-transformer-sharp: ^2.5.2 => 2.5.2 
  npmGlobalPackages:
    gatsby-cli: 2.12.34

```

**Comment by madalynrose** [comment]
Hi there @MattixNow! I'm trying to understand the root of your issue. Could you explain what you mean by "bottom bar" and which output you are unable to see? I believe the status bar has always existed and want to make sure I fully understand the bug you are reporting.

**Comment by MattixNow** [comment]
Hey, thanks for your reply !

The bottom bar is this
```bash
————————————————————————————————————————————————————————————————————————————————————————
5 pages                              SUCCESS                              gatsby-starter-default
```
The bottom bar isn't displaying in all of my gatsby project (displayed on the gatsby-starter-default, but not on the gatsby-starter-hello-world). In vscode this output take a lot of space when the integrated terminal is opened (the user have to change the height of the integrated terminal) and the informations which are display are not useful (number of pages, build status, project name) knowing that the information relating to the built is displayed just above.

Finally i think it's not user friendly, it's an accessibility issue

**Comment by LekoArts** [comment]
The status bar is in our CLI since a year already (https://github.com/gatsbyjs/gatsby/pull/14973) so this is nothing new. It'll display in all projects and so far we haven't heard any complaints.

What are the accessibility issues?

**Comment by MattixNow** [comment]
![Annotation 2020-06-02 174919](https://user-images.githubusercontent.com/32040951/83541614-28b5be00-a4fa-11ea-8fa4-750fb37a2264.jpg)
You can see the wide space the bottom bar takes on my screen, this is why i think it's an accessibility issue

**Comment by LekoArts** [comment]
Thanks for the follow-up!

I think it's debatable if the information is useless or not, in our opinion it's useful and something we want to keep. In my terminal it doesn't have the height you're showing so you might check with https://github.com/vadimdemedes/ink if your CLI is having issues displaying the spaces correctly.

However, I found that the develop bar doesn't always show, so I created an issue for that: https://github.com/gatsbyjs/gatsby/issues/24736

Thanks for the issue!
